src/App.jsx

```diff
@@ -14,70 +16,66 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [nameGood, setNameGoog] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      {nameGood ? (
+        <h1 className="title is-flex is-align-items-center">
+          {`${nameGood} `}
+          is selected
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => {
+              setNameGoog('');
+            }}
+          />
+        </h1>
+      ) : (
+        <h1 className="title is-flex is-align-items-center">
+          No goods selected
+        </h1>
+      )}
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
+      <table className="table">
+        <tbody>
+          {goods.map(good => (
+            <tr
+              key={good}
+              data-cy="Good"
+              className={cn({
+                'has-background-success-light': nameGood === good,
+              })}
             >
-              +
-            </button>
-          </td>
+              <td>
+                <button
+                  data-cy={nameGood === good ? 'RemoveButton' : 'AddButton'}
+                  type="button"
+                  className={cn('button', { 'is-info': nameGood === good })}
+                  onClick={() => {
+                    if (nameGood === good) {
+                      setNameGoog('');
+                    } else {
+                      setNameGoog(good);
+                    }
+                  }}
+                >
+                  {nameGood === good ? '-' : '+'}
+                </button>
+              </td>
```

Comment:
if we change almost everything depends on condition `nameGood === good`
better to write:
```
{nameGood === good ? (
  <tr>... </tr>
) : (
   <tr> ... </tr>
)}
```